src/transit_equity/utils/db_helpers.py

```diff
@@ -0,0 +1,51 @@
+from sqlalchemy import MetaData, Engine
+from sqlalchemy.ext.automap import automap_base, AutomapBase
+
+
+def get_automap_base_with_views(engine: Engine, schema: str) -> AutomapBase:
+    '''
+    Returns an AutomapBase object that also provides access to views of a schema
+
+    Parameters
+    ----------
+    engine : sqlalchemy.Engine
+        Engine that is already connected to a database
+    schema : str
+        Name of schema in the database for which we want to get an AutomapBase object
+
+    Returns
+    -------
+    AutomapBase
+        A declarative automap base for the schema of the database
+
+    Examples
+    --------
+    >>> from sqlalchemy import create_engine
+    >>> engine = create_engine('postgresql://user:password@localhost:5432/dbname'))
+    >>> Base = get_automap_base_with_views(engine=engine, schema='orca')
+    >>> print(type(Base))
+    <class 'sqlalchemy.ext.automap.AutomapBase'>
+    '''
+    metadata = MetaData()
+    metadata.reflect(bind=engine, views=True, schema=schema)
+    Base: AutomapBase = automap_base(metadata=metadata)
+    Base.prepare()
+    return Base
+
+# Run this from the root to test it
+if __name__=='__main__':
```

Comment:
I have left it here for temporary testing until we are able to install this as a package.
Will get rid of it eventually. @bhazelton